packages/builder/src/builder.js

```diff
@@ -525,14 +473,13 @@ export default class Builder {
       }
 
       if (indicatorPath) {
-        templatesFiles.push({
+        templateFiles.push({
           src: indicatorPath,
           dst: 'loading.html',
           custom: customIndicator,
           options: this.options.loadingIndicator
         })
       } else {
-        /* istanbul ignore next */
         // eslint-disable-next-line no-console
         console.error(
```

Comment:
Use `consola`?

Reply:
A fact: All console logs with `nuxt` cli will be redirected to consola's corresponding logger. But would be good idea for consistency and cleaner code.

Reply:
@ricardogobbosouza I may work on log migration for all nuxt code in another pr.

Reply:
You're write, will change it